lib/tasks/pulfalight.rake

```diff
@@ -294,8 +290,13 @@ namespace :pulfalight do
     file_paths = Dir.glob(glob_pattern)
 
     file_paths.each do |file_path|
+      # Don't index full versions of seed files if given argument.
+      next if file_path.include?(".processed") && file_path.include?("MC221")
+      # Index all of MC221 - we have several tests for it.
+      next if File.exist?(file_path.gsub(".EAD", ".processed.EAD")) && !file_path.include?("MC221")
```

Comment:
question: why are certain fixtures considered "processed"? Where does that come from?